src/flow-worker.c

```diff
@@ -67,14 +70,17 @@ typedef struct FlowWorkerThreadData_ {
  *
  *  Handle flow creation/lookup
  */
-static inline TmEcode FlowUpdate(Packet *p)
+static inline TmEcode FlowUpdate(Packet *p, ThreadVars *tv, FlowWorkerThreadData *fw)
 {
     FlowHandlePacketUpdate(p->flow, p);
 
     int state = SC_ATOMIC_GET(p->flow->flow_state);
     switch (state) {
         case FLOW_STATE_CAPTURE_BYPASSED:
+            return TM_ECODE_DONE;
```

Comment:
if we'd count these as well separately we'd get a sense for latency for applying offload?

Reply:
Yes indeed.